packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -1100,7 +1100,7 @@ export const studioLocaleStrings = defineLocalesResources('studio', {
   /** Title of the alert for studio users when packages in their studio are out-of-date */
   'package-version.new-package-available.title': 'Sanity Studio is ready to update!',
   /** Label for action to manage members of the current studio project */
-  'presence.action.manage-members': 'Manage members',
+  'presence.action.manage-members': 'Invite collaborators',
```

Comment:
Suggest we call this "Invite members" so we are consistent with the user dropdown item.

Also wondering what we do with the i18n – do we need to update this in the language packages too?

Reply:
String is updated - no idea what the process is for inputing translations

Reply:
Check with someone from the Studio team or @runeb. They are located in https://github.com/sanity-io/locales, but I'm not sure about the process either.